ed25519_public_key.go

```diff
@@ -148,6 +148,10 @@ func (pk _Ed25519PublicKey) _ToProtoKey() *services.Key {
 	return &services.Key{Key: &services.Key_Ed25519{Ed25519: pk.keyData}}
 }
 
+func (pk _Ed25519PublicKey) String() string {
```

Comment:
The added String() methods are so that these structs implement `Key` interface, as thy should